tasks/transform_conformance/babel.snap.md

```diff
@@ -608,6 +607,10 @@ Passed: 327/1369
 * transparent-expr-wrappers/ts-as-member-expression/input.ts
 * transparent-expr-wrappers/ts-parenthesized-expression-member-call/input.ts
 
+# babel-plugin-transform-json-strings (2/4)
```

Comment:
The test case crashed because we need to escape `\u2028\u2029` at `plugin-transform-json-strings`. I can implement it at other pr.

Reply:
I think we can merge this, but we will need to create an issue to track it

Reply:
I already implemen it, i will open a new pr.